activerecord/lib/active_record/model_schema.rb

```diff
@@ -499,6 +500,30 @@ def compute_table_name
             base.table_name
           end
         end
+
+        def warn_if_deprecated_type(column)
+          return if attributes_to_define_after_schema_loads.key?(column.name)
+          return unless column.respond_to?(:oid)
+          if column.array?
+            array_arguments = ", array: true"
+          else
+            array_arguments = ""
+          end
+          if column.sql_type.start_with?("interval")
```

Comment:
@sgrif, I've noticed that you've removed this deprecation warning for points in [`b347156`](https://github.com/rails/rails/commit/b347156b15aea7f6229203e0cc1a58937ecb8e93#diff-6c8fd220c92c2b25aa29891b59d00c04), but it's not clear to me whether I should to remove it too or not. I decided to keep it for now. Please suggest the proper solution here.